api_docs.yml

```diff
@@ -2572,6 +2572,182 @@ paths:
         500:
           description: "Internal Server Error"
 
+  "/projects/{project_id}/users":
+    post:
+      tags:
+        - projects
+      consumes:
+        - application/json
+      description: "Add a user to a project, role can be either 'owner' or 'admin' or 'member'"
```

Comment:
I think the owner role should only be assigned at project creation

Reply:
Oh yeah , that's just in API docs,, but we don't accept owners